app/models/manageiq/providers/inventory/persister/builder/infra_manager.rb

```diff
@@ -84,8 +84,8 @@ def vm_and_template_ems_custom_fields
 
           add_properties(
             :model_class                  => ::CustomAttribute,
-            :manager_ref                  => %i(name),
-            :parent_inventory_collections => %i(vms)
+            :manager_ref                  => %i(resource name),
+            :parent_inventory_collections => %i(vms miq_templates)
```

Comment:
This collection (vm_and_template_ems_custom_fields) is used by ovirt and exhibits the same bug, I am going to follow-up with a refactoring to get them both onto the same inventory_collection but want to keep this clean of refactorings for backport